src/components/PrinterModal.vue

```diff
@@ -3,7 +3,7 @@
     <traction-button
       id="printLabels"
       v-traction-modal.printerModal
-      variant="primary"
```

Comment:
Lots of this kind of thing. Variant didn't bother me too much, but I wanted the theme names to make sense in the context of a button. What is a successful button?